internal/commands/artifacts.go

```diff
@@ -0,0 +1,14 @@
+package commands
+
+import (
+	"github.com/Khan/genqlient/graphql"
+	"github.com/massdriver-cloud/mass/internal/api"
+)
+
+func GetAllArtifacts(client graphql.Client, orgID string, name string) ([]api.Artifact, error) {
```

Comment:
ListArtifacts to stay consistent 

logic in your cobra commands should be moved here + tests

Goal is to make commands/ testable versions of the cobra commands but without any terminal I/O flag parsing outputting to files etc